edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,148 @@
+{% load i18n %}
+<div class="sequence proctored-exam instructions" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Awaiting Proctoring Software Installation and Setup
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      You have chosen to take {{display_name}} as a proctored exam. You should be directed to a new window
+      with installation and setup instructions. You can also <a href="{{software_download_url}}" target="_blank">open the installation window directly.</a>
+    {% endblocktrans %}
+  </p>
+  <div class="proctored-exam-message">
+    <h3>
+    {% blocktrans %}
+      Here is your unique exam code. You will be asked for it during the setup.
```

Comment:
Delete "the": "You will be asked for it during setup."
Hopefully this will allow everything to fit on one line?